int64 test
---
bool object_detected